cvat-ui/src/components/projects-page/styles.scss

```diff
@@ -71,9 +71,20 @@
     justify-content: center;
 }
 
-.cvat-projects-project-item-title,
```

Comment:
IMHO, it is a little bit confusing that the title has no cursor pointer

Reply:
Fixed